GFS_layer/GFS_typedefs.F90

```diff
@@ -680,6 +680,9 @@ module GFS_typedefs
     logical              :: do_z0_hwrf17    !< flag for using z0 scheme in 2017 HWRF (kgao)
     logical              :: do_z0_hwrf17_hwonly !< flag for using z0 scheme in 2017 HWRF only under high wind (kgao)
     real(kind=kind_phys) :: wind_th_hwrf    !< wind speed threshold when z0 level off as in HWRF (kgao)
+    real(kind=kind_phys) :: alpha_stable    !< description TBD
+    real(kind=kind_phys) :: alpha_unstable  !< description TBD
```

Comment:
Can you add these descriptions, and perhaps note these were added by Sofar and the date.